x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.tsx

```diff
@@ -108,6 +114,19 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
     [onLoading, unsnoozeRule, item, onRuleChanged, toasts, onClose]
   );
 
+  const cloneRuleInternal = useCallback(
+    async (ruleId: string) => {
+      try {
+        const cloneItem = await cloneRule(item.id);
```

Comment:
As you are not using the ruleId set as param it looks like you don't need to do the binding in line 236. Also, `item` here is exactly the alert which opened the "collapsed item actions" so you should be fine although it's best if you could please check 😓

Reply:
yes I think you right because I move the the popover state outside of it